.github/workflows/windows.yml

```diff
@@ -81,6 +84,28 @@ jobs:
           path: 'openvino_contrib'
           ref: 'master'
 
+      - name: Setup Python ${{ env.PYTHON_VERSION }}
```

Comment:
Should it be a part of the action?

Reply:
I don't think so (e.g. on Linux we already have Python in Docker)

Reply:
why it is needed then here? )